src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -1299,13 +1299,11 @@ class TcgStatics {
     _delegate.delayed {
```

Comment:
Does this static work correctly if a supporter is not used from hand, such as being used via an attack for example? The Supporter should work if it is played as the effect of an attack, played from the discard (don't know if an effect like this exists) or if you play the effect of a Supporter from the opponent's hand, right?

Reply:
Didn't check for this, the static already existed as this. Will check it in a bit.

Reply:
Thinking about it, it could reduce the "max supporters from hand per turn" limit. Though it'd still cause some weird interactions with Lt. Surge's Strategy.

Reply:
> Does this static work correctly if a supporter is not used from hand, such as being used via an attack for example?

Finally got to test this a bit. And sadly, I don't see a clean way to solve this as of now. Every single attack preventing the play of trainers (or supporters in specific) currently prevent the effect from being done. But in my opinion a change needs to be done on the side of the attack/ability replicating the supporter.

Currently, they're discarded and played as themselves (being considered a trainer effect, or whatever indicated in the statics of the trainer cards themselves); when in reality only the effect (and needed checks/costs ofc) should be used, after being converted into an attack-sourced effect. A mimikyu impersonating guzma should be able to target a Omega Barrier Pokémon and switch it without trouble, unlike the regular supporter.

Will merge this for now, as the commited fixes are still relevant. And added a flag so the supporters are played at least. But it's still not a perfect fix.

Reply:
Cards that I will not add a flag check for in this commit everywhere, but may need it:
* Smeargle (Undaunted, Call of Legends)
* Glalie (Mysterious Treasures)